### DISCUSSION TITLE
Search on enter v-data-table

### DISCUSSION THREAD

**Discussion by Ce11an** [vuetifyjs/vuetify#14680] [Q&A] [answered]
Hi everyone,

I'm new to Vue.js and I have been enjoying using Vuetify to build small projects!

I was wondering if there a way of using the search prop to only search on keyup.enter, instead of searching on every character typed? Any suggestions would be welcomed!

Thanks in advance!

**Comment by Ce11an** [comment, marked as answer]
For now, I have used a watch on the input with a debounce function so that a search only performs after 600ms have passed without any input. Working well so far!

**Reply by seppeclijsters** [reply]
Smart idea! Thanks
